versions/h-/hareflow.json

```diff
@@ -0,0 +1,9 @@
+{
+  "versions": [
+    {
+      "git-tree": "8a7cda35bd1171a017776be1aa826fb2e57c7d5c",

```

Comment:
I don't think this is the latest: `git rev-parse HEAD:ports/hareflow/` gives me `78f24783515b0f4b2aca4f7d5b80aa600eacf110`.

Reply:
je suis confus, je l'avais re-roulé